### ISSUE TITLE
Port 80 already in use

### ISSUE THREAD

**Issue by TathagataRoy1278** [MSU-NatSci/DomainAccessibilityAudit#34]
I run 
`sudo docker-compose up -d`

and it throws the error - 
```
domainaccessibilityaudit_mongodb_1 is up-to-date
Starting accessibility_audit ... 
Starting accessibility_audit ... error

ERROR: for accessibility_audit  Cannot start service accessibility_audit: driver failed programming external connectivity on endpoint accessibility_audit (485d0fd6c8a014ea59922afc03f6534e4ebd59795f61c7255a451056eadf606c): Error starting userland proxy: listen tcp4 0.0.0.0:80: bind: address already in use

ERROR: for accessibility_audit  Cannot start service accessibility_audit: driver failed programming external connectivity on endpoint accessibility_audit (485d0fd6c8a014ea59922afc03f6534e4ebd59795f61c7255a451056eadf606c): Error starting userland proxy: listen tcp4 0.0.0.0:80: bind: address already in use
ERROR: Encountered errors while bringing up the project.
```

I understand that it says port 80 is in use, and it is indeed in use by the apache server. Since I am not at all familiar with docker, could  someone tell me how I can start it on another port?

Thank you !

**Comment by TathagataRoy1278** [comment]
Im an idiot.
Looked at the `docker-compose.yml` and saw a `$PRODUCTION_PORT`, defined that.
After doing that `docker-compose up -d` runs as 
```
Starting domainaccessibilityaudit_mongodb_1 ... done
Recreating accessibility_audit              ... done
```
